### PR TITLE
Add some functiolaties in List Pendings, without logic

### PR DESCRIPTION
**Pull request by alejandrogon1418**
Added the panel to Swing and other things

### CODE REVIEW

File: final-progII/src/presentation/Menu.java [hunk 1]
```diff
@@ -151,7 +149,7 @@ public void actionPerformed(ActionEvent e) {
 		
 		listPendings_menuItem.addActionListener(new ActionListener() {
 			public void actionPerformed(ActionEvent e) {
-				master__cardLayout.show(master__panel, "LIST_STUDENTS_PENDINGS");
+				master__cardLayout.show(master__panel, "name_120454282608500");
```
Comment (nicocadq) [inline final-progII/src/presentation/Menu.java:152]: Why did you change the name of the panel?

File: final-progII/src/presentation/Screen.java [hunk 2]
```diff
@@ -1210,6 +1212,11 @@ public void actionPerformed(ActionEvent arg0) {
 		JButton deleteAbsence__btnNewButton = new JButton("Delete");
 		deleteAbsence__btnNewButton.setBounds(10, 283, 89, 23);
 		infoConsultAbsences__panel.add(deleteAbsence__btnNewButton);
+		
+		idtDelete__textField = new JTextField();
+		idtDelete__textField.setBounds(10, 262, 86, 20);
+		infoConsultAbsences__panel.add(idtDelete__textField);
+		idtDelete__textField.setColumns(10);
```
Comment (nicocadq) [inline final-progII/src/presentation/Screen.java:1219]: I don't know what is `idtDelete`, It's just a suggestion but maybe you could rename this text field to be more readable.

File: final-progII/src/presentation/Menu.java (hunk 1 above)
Comment (alejandrogon1418) [reply final-progII/src/presentation/Menu.java:152]: because in the class screen was named like that.

File: final-progII/src/presentation/Screen.java (hunk 2 above)
Comment (alejandrogon1418) [reply final-progII/src/presentation/Screen.java:1219]: I dont know either. Maybe I'll change it
